### PR TITLE
:gear: :hook:  Refactor Statefull to stateless

### PR DESCRIPTION
**Pull request by SimonGideon**
## In this PR I implemented the following:

- [ ] Use React hooks.
- [ ] Added a calculator favicon to the App

:slightly_smiling_face: Thanks for the review :+1:

### CODE REVIEW

File: src/components/Calculator.js [hunk 1]
```diff
@@ -1,94 +1,88 @@
-import React from 'react';
+import React, { useState } from 'react';
 /* eslint-disable react/destructuring-assignment */
```
Comment (muneebulrehman) [inline src/components/Calculator.js:2]: It is not recommended you disable linters. Kindly remove this line which disables linters.

**Review by muneebulrehman — CHANGES_REQUESTED** [review]
Hi @SimonGideon , 

Good job so far! I see you put a lot of effort into this project. However, there are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- No linter errors
- Descriptive PR
- Working project

## Required Changes ♻️
_Check the comments under the review._

- [ ] Your app.js is still using the Class based component. Kindly use the Functional component for the same.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/Calculator.js (hunk 1 above)
Comment (SimonGideon) [reply src/components/Calculator.js]: @muneebulrehman Thanks for the review :slightly_smiling_face: . The requested changes have been implemented on the proceeding commit. :100:

**Review by Rustamxon7 — APPROVED** [review]
Hi @SimonGideon, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Developer-512px-10

## Highlights
1. No Linters errors 💯
2. All requirements are complete 💯

## Optional suggestions
Kindly add MIT license to your project. 
Also, I suggest you remove this line from your readme file:

Screenshot 2022-12-01 at 12 54 34

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**